scripts/events_module/event_filters.py

```diff
@@ -380,3 +381,148 @@ def _check_cat_gender(cat, genders: list) -> bool:
         return True
 
     return False
+
+
+def cat_for_event(constraint_dict: dict, possible_cats: list, comparison_cat=None):
+    """
+    Checks the given cat list against constraint_dict to find any eligible cats.
+    Returns a single cat ID chosen from eligible cats.
+    :param constraint_dict: Can include age, status, skill, trait, and backstory lists
+    :param possible_cats: List of possible cat objects
+    :param comparison_cat: If you need to search for cats with a specific relationship status, then include a comparison
+     cat. Keep in mind that this will search for a possible cat with the given relationship toward comparison cat.
+    """
+    # gather funcs to use
+    funct_dict = {
```

Comment:
```suggestion
    func_dict = {
```
Nitpicky thing, but the shorthand term for "function" is "func". I've been driven mad enough by `cur_screen` (i am convinced it should be curr_screen, damn it!), so I'm gonna be annoying here and get it right before it gets locked in